pages/index.js

```diff
@@ -3,11 +3,19 @@ import Navigation from "@/components/Navigation/Navigation";
 import useSWR from "swr";
 import SearchBar from "@/components/SearchBar/SearchBar";
 import { useState, useEffect } from "react";
+import FilterBar from "@/components/FilterBar/FilterBar";
 
 export default function Home() {
-  const { data, error } = useSWR("https://www.tagesschau.de/api2/news");
+  const [region, setRegion] = useState("");
+  const [ressort, setRessort] = useState("");
   const [filteredArticles, setFilteredArticles] = useState([]);
 
+  const { data, error } = useSWR(
+    `https://www.tagesschau.de/api2/news?${region ? `regions=${region}` : ""}&${
+      ressort ? `ressort=${ressort}` : ""
+    }`
+  );
+
```

Comment:
Currently, the base url shows both characters `?` and `&` even if you don't have any parameters. In order to keep it clean, you can construct it somewhat differently like this:
```suggestion
  const baseUrl = "https://www.tagesschau.de/api2/news";
  const params = [];
  
  if (region) {
    params.push(`regions=${region}`);
  }
  
  if (ressort) {
    params.push(`ressort=${ressort}`);
  }

  const finalUrl = `${baseUrl}${params.length ? `?${params.join('&')}` : ''}`;
  const { data, error } = useSWR(finalUrl);
```

Reply:
Thanks for the suggestion! I'll implement this cleaner URL approach. 👍